src/pesterTestController.ts

```diff
@@ -473,6 +473,15 @@ export class PesterTestController implements Disposable {
 			await this.powerShellExtensionClient.GetVersionDetails()
 		}
 
+        const pesterModulePath = workspace
+            .getConfiguration('pester')
+            .get<string>('pesterModulePath')
+
+        if (pesterModulePath) {
+            scriptArgs.push('-PesterModulePath')
+            scriptArgs.push(`"${pesterModulePath}"`)
```

Comment:
If this is a relative path I need to prefix it with the project path. Can you point me to how I get the current project path (docs or code snippet)?

Reply:
This is tricky, because you can have a multi-workspace root
https://github.com/microsoft/vscode/wiki/Adopting-Multi-Root-Workspace-APIs